baseframe/static/js/baseframe-material.js

```diff
@@ -249,6 +249,22 @@ window.Baseframe.Forms = {
         }
       }
     })
+  },
+  showValidationErrors: function(errors) {
+    Object.keys(errors).forEach(function(fieldName) {
+      if (Object.prototype.toString.call(errors[fieldName]) === '[object Array]') {
```

Comment:
Specify the polyfill for `Array.isArray` in `baseframe.js`, so it can reused.

Reply:
Only IE8 doesn't support `Array.isArray` so I guess we could use it.